std_msgs/Float64MultiArray q
bool async
---
bool result
